src/Microsoft.ApiDesignGuidelines.Analyzers/UnitTests/OverrideMethodsOnComparableTypesTests.Fixer.cs

```diff
@@ -119,29 +210,132 @@ public int CompareTo(object obj)
 
     public override bool Equals(object obj)
     {
+        if (ReferenceEquals(obj, null))
+        {
+            return false;
+        }
+
         throw new NotImplementedException();
     }
 
     public static bool operator ==(A left, A right)
     {
-        throw new NotImplementedException();
+        if (ReferenceEquals(left, right))
```

Comment:
Added a comment here: https://github.com/dotnet/roslyn-analyzers/issues/791#issuecomment-229736838

You can probably remove this if statement and improve Equals implementation to upfront compare `ReferenceEquals(this, other)` and return true.